### DISCUSSION TITLE
GitHub gives wrong URL for repository

### DISCUSSION THREAD

**Discussion by roamn2** [community/community#37521] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello there. 
I have the following problem:  
I have made a repository with .html-files and I wanted to put it on GitHub Pages, so  I made the repository name "https://roamn.github.io" but GitHub changes it to "https://github.com/roamn2/roamn.github.io.git" which makes it  unaccessable.   

Please help me with this problem.  
Thank you.

**Comment by airtower-luna** [comment, marked as answer]
Your `github.io` domain is based on your username, so making a user site will only work with a repository called `roamn2.github.io` (unless you change your username or make an organization with the name you want). Pages sites for any other repository will appear in subdirectories, unless you give them custom domains.

**Reply by roamn2** [reply]
Hello.
No it still does not work.
<img width="1012" alt="img" src="https://user-images.githubusercontent.com/99689942/198332022-ece21c63-1f6f-44fe-84d6-5a0f23bbfa0c.png">

**Reply by airtower-luna** [reply]
That's a clone URL, not one for Pages. Also I can see your [`roamn2.github.io` repo](https://github.com/roamn2/roamn2.github.io) is empty, so there can't be anything published.

**Reply by roamn2** [reply]
I forgot that. 
Now I added some files (html, png, md).

Thank you very much.
Have a nice day. ☺

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :slightly_smiling_face:
